snippets/freeipa_register.erb

```diff
@@ -24,6 +24,9 @@ name: freeipa_register
 #
 #   freeipa_opts                Additional options to pass directly to installer
 #
+
+/usr/sbin/sshd-keygen
```

Comment:
I don't think it'd be appropriate here to store the details of what host keys should be present, leave that to the OS tools if at all possible. Key types, lengths and locations will vary between OSes and versions, hardcoding that would cause it to be obsolete quickly.